converter/src/main/java/gov/cms/qpp/conversion/validate/CpcPerformancePeriodValidation.java

```diff
@@ -24,7 +22,9 @@
 	@Override
 	protected void internalValidateSingleNode(Node node) {
 		check(node)
-			.valueIs(PERFORMANCE_START_JAN12017, ReportingParametersActDecoder.PERFORMANCE_START, "20170101")
-			.valueIs(PERFORMANCE_END_DEC312017, ReportingParametersActDecoder.PERFORMANCE_END, "20171231");
+			.valueIs(ErrorCode.CPC_PERFORMANCE_PERIOD_START_JAN12017, 
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this 121 characters long line (which is greater than 120 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)